chore(hybrid-cloud): Add generic docs for SCIM/SAML when renaming an org. (#5327)
getsentry/sentry-docs: src/docs/product/accounts/sso/azure-sso.mdx

## Installation

<Alert level="warning">

> If you change your organization slug, you'll need to make the same update in the steps where you enter the SAML configuration values.

Reviewer: @dashed - I was indicating that the change I suggested here should be applied in all the files.

Author: @imatwawana yep! I'm applying them everywhere 👍

> If you rename your organization's slug, you will need to replace the previous organization slug with the new slug name in the relevant steps where you enter the SAML configuration values.

Reviewer: Edited the wording for style and concision and to remove what seems like some redundancy in the wording. Same edit applies to all of the notes.

```suggestion
If you change your organization slug, you'll need to make the same update in the steps where you enter the SAML configuration values.
```
